doc/examples/sprites/lib/main.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:flame/flame.dart';
+import 'package:flutter/material.dart';
+import 'package:flame/components/component.dart';
+import 'package:flame/game.dart';
+
+void main() async {
+  final Size size = await Flame.util.initialDimensions();
+  final game = MyGame(size);
+  runApp(game.widget);
+}
+
+class MyGame extends BaseGame {
+  MyGame(Size screenSize) {
+    size = screenSize;
+  }
+
+  @override
+  void onAttach() {
+    super.onAttach();
+
+    initSprites();
+  }
+
+  void initSprites() async {
+    final r = Random();
+    try {
+      List.generate(500, (i) => SpriteComponent.square(32, 'test.png')).forEach((sprite) {
+        sprite.x = r.nextInt(size.width.toInt()).toDouble();
+        sprite.y = r.nextInt(size.height.toInt()).toDouble();
+        add(sprite);
+      });
+    } catch (e) {
+      debugPrint(e.toString());
+    }
```

Comment:
is there any specific exception you were looking for here? overall I'd rather not have the try catch and let it blow (if anything ever happens). this is just an example app, but always good to fail fast and fix. also makes the code a tad bit cleaner

Reply:
i think i had some initial problem loading assets in the example when it was in the Example folder, but i'll remove it